packages/components/src/AutoCompleteInput.tsx

```diff
@@ -49,6 +49,7 @@ interface AutoCompleteInputProps {
   spellCheck: boolean;
   onEnter(): void;
   noMatchText: string;
+  'data-testid'?: string;
```

Comment:
I think these data-testid bits are showing up in your review because you based this change off your existing data-testid branch instead of off of the latest main.
For this change you should be able to just rebase your changes off the latest main; in the future, start a new branch for each change instead of basing off your old branch (unless it is dependent on your old branch).